client/src/components/mixins/name-request-mixin.ts

```diff
@@ -143,16 +143,6 @@ export default class NameRequestMixin extends Vue {
     newRequestModule.mutateSubmissionTabNumber(this.submissionTabNumber - 1)
   }
 
-  private isloadingSubmission: boolean = false
```

Comment:
It looks like this should work but it doesn't. This file is a mixin, so all the methods are inserted into `this` for the pages. However I found that `isLoadingSubmission` was always undefined in the pages, so the button loading state never worked.